src/sidebar/panel.js

```diff
@@ -30,11 +66,52 @@ const quill = new Quill('#editor', {
   theme: 'snow',
   placeholder: browser.i18n.getMessage('emptyPlaceHolder'),
   modules: {
+    keyboard: {
+      bindings: bindings
+    },
     toolbar: '#toolbar'
   },
   formats: formats // enabled formats, see https://github.com/quilljs/quill/issues/1108
 });
 
+
+function determineOS() {
+  var OSName = "";
+  
+  if (navigator.appVersion.indexOf("Win") != -1)
+    OSName = "win";
+  if (navigator.appVersion.indexOf("Mac") != -1)
+    OSName = "mac";
+  if (navigator.appVersion.indexOf("Linux") != -1)
+    OSName = "linux";
+  
+  return OSName;
+}
+
+const userOS = determineOS();
+var userOSKey = '';
+
+if (userOS === 'mac')
+  userOSKey = '⌘';
+else
+  userOSKey = 'Ctrl';
+
+var size = document.getElementsByClassName("ql-size")[0],
+    bold = document.getElementsByClassName("ql-bold")[0],
+    italic = document.getElementsByClassName("ql-italic")[0],
+    strike = document.getElementsByClassName("ql-strike")[0],
+    ordered = document.getElementsByClassName("ql-list")[0],
+    bullet = document.getElementsByClassName("ql-list")[1];
+
+// Setting button titles in place of tooltips
+size.title = "Font size",
+  bold.title = "Bold (" + userOSKey + "+B)",
+  italic.title = "Italic (" +userOSKey + "+I)",
+  strike.title = "Strikethrough (" + userOSKey + "+Shift+" + bindings.strike.key + ")",
+  ordered.title = "Numbered list (" + userOSKey + "+Shift+" + bindings.ordered.key + ")",
+  bullet.title = "Bulleted list (" + userOSKey + "+Shift+" + bindings.bullet.key + ")";
```

Comment:
All these need to be translated, see example above, something like `placeholder: browser.i18n.getMessage('emptyPlaceHolder'),`